### ISSUE TITLE
Could not find module './web_modules/import-map.json'

### ISSUE THREAD

**Issue by Tohaker** [pikapkg/create-snowpack-app#34]
Upon running `npx create-snowpack-app new-dir --template @snowpack/app-template-react-typescript` and then running `npm start` in `new-dir`, the following error occurs:

```
> @ start /workspaces/snowpack-test/new-dir
> snowpack dev

Snowpack Dev Server (Beta)
NOTE: Still experimental, default behavior may change.
Starting up...

Error: Cannot find module '/workspaces/snowpack-test/new-dir/web_modules/import-map.json'
Require stack:
- /workspaces/snowpack-test/new-dir/node_modules/snowpack/dist-node/index.js
- /workspaces/snowpack-test/new-dir/node_modules/snowpack/dist-node/index.bin.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at command$1 (/workspaces/snowpack-test/new-dir/node_modules/snowpack/dist-node/index.js:993:31)
    at cli (/workspaces/snowpack-test/new-dir/node_modules/snowpack/dist-node/index.js:3159:11)
```

Looks like the web_modules import map isn't being created, but the docs say this will be done automatically upon installation

**Comment by FredKSchott** [comment]
Yup, definitely should be done automatically. Do you have the output from running `npx create-snowpack-app new-dir --template @snowpack/app-template-react-typescript` available to share? That's the stage that install should have been run.

**Comment by Tohaker** [comment]
```npx create-snowpack-app new-dir --template @snowpack/app-template-react-typescript
npx: installed 36 in 2.69s

  - Using template @snowpack/app-template-react-typescript
  - Creating a new project in /workspaces/snowpack-test/new-dir
  - Installing package dependencies. This might take a couple of minutes.


> deasync@0.1.20 install /workspaces/snowpack-test/new-dir/node_modules/deasync
> node ./build.js

`linux-x64-node-12` exists; testing
Binary is fine; exiting

> core-js@2.6.11 postinstall /workspaces/snowpack-test/new-dir/node_modules/babel-runtime/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> core-js@3.6.5 postinstall /workspaces/snowpack-test/new-dir/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> core-js-pure@3.6.5 postinstall /workspaces/snowpack-test/new-dir/node_modules/core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"


> core-js@2.6.11 postinstall /workspaces/snowpack-test/new-dir/node_modules/parcel-bundler/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"


> parcel-bundler@1.12.4 postinstall /workspaces/snowpack-test/new-dir/node_modules/parcel-bundler
> node -e "console.log('\u001b[35m\u001b[1mLove Parcel? You can now donate to our open collective:\u001b[22m\u001b[39m\n > \u001b[34mhttps://opencollective.com/parcel/donate\u001b[0m')"

Love Parcel? You can now donate to our open collective:
 > https://opencollective.com/parcel/donate
added 1491 packages from 720 contributors and audited 286030 packages in 515.023s

46 packages are looking for funding
  run `npm fund` for details

found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details

  - Initializing git repo.

  - Success!

Quickstart:

  cd new-dir
  npm start

All Commands:

  npm install
  Install all dependencies (npm + snowpack).
  We already ran this one for you.
  npm start
  Starts the development server.
  npm run build
  Bundles the app into static files for production.
  npm test
  Starts the test runner.
```

**Comment by Tohaker** [comment]
I should note that running `npm run prepare` fixes this, though I expected it to run after running the creation command

**Comment by FredKSchott** [comment]
Yup, no idea why it didn't run originally. Adding a check now to the installer that if web_modules directory doesn't exist, just run it manually before finishing.

**Comment by FredKSchott** [comment]
resolved by c741106
